react/features/base/util/roomnameGenerator.js

```diff
@@ -0,0 +1,196 @@
+    'Anxious', 'Appalling', 'Apparent', 'Astonishing', 'Attractive', 'Awesome',
+    'Baby', 'Bad', 'Beautiful', 'Benign', 'Big', 'Bitter', 'Blind', 'Blue',
+    'Bold', 'Brave', 'Bright', 'Brisk', 'Calm', 'Camouflaged', 'Casual',
+    'Cautious', 'Choppy', 'Chosen', 'Clever', 'Cold', 'Cool', 'Crawly',
+    'Crazy', 'Creepy', 'Cruel', 'Curious', 'Cynical', 'Dangerous', 'Dark',
+    'Delicate', 'Desperate', 'Difficult', 'Discreet', 'Disguised', 'Dizzy',
+    'Dumb', 'Eager', 'Easy', 'Edgy', 'Electric', 'Elegant', 'Emancipated',
+    'Enormous', 'Euphoric', 'Evil', 'Fast', 'Ferocious', 'Fierce', 'Fine',
+    'Flawed', 'Flying', 'Foolish', 'Foxy', 'Freezing', 'Funny', 'Furious',
+    'Gentle', 'Glorious', 'Golden', 'Good', 'Green', 'Green', 'Guilty',
+    'Hairy', 'Happy', 'Hard', 'Hasty', 'Hazy', 'Heroic', 'Hostile', 'Hot',
+    'Humble', 'Humongous', 'Humorous', 'Hysterical', 'Idealistic', 'Ignorant',
+    'Immense', 'Impartial', 'Impolite', 'Indifferent', 'Infuriated',
+    'Insightful', 'Intense', 'Interesting', 'Intimidated', 'Intriguing',
+    'Jealous', 'Jolly', 'Jovial', 'Jumpy', 'Kind', 'Laughing', 'Lazy', 'Liquid',
+    'Lonely', 'Longing', 'Loud', 'Loving', 'Loyal', 'Macabre', 'Mad', 'Magical',
+    'Magnificent', 'Malevolent', 'Medieval', 'Memorable', 'Mere', 'Merry',
+    'Mighty', 'Mischievous', 'Miserable', 'Modified', 'Moody', 'Most',
+    'Mysterious', 'Mystical', 'Needy', 'Nervous', 'Nice', 'Objective',
+    'Obnoxious', 'Obsessive', 'Obvious', 'Opinionated', 'Orange', 'Painful',
+    'Passionate', 'Perfect', 'Pink', 'Playful', 'Poisonous', 'Polite', 'Poor',
+    'Popular', 'Powerful', 'Precise', 'Preserved', 'Pretty', 'Purple', 'Quick',
+    'Quiet', 'Random', 'Rapid', 'Rare', 'Real', 'Reassuring', 'Reckless', 'Red',
+    'Regular', 'Remorseful', 'Responsible', 'Rich', 'Rude', 'Ruthless', 'Sad',
+    'Scared', 'Scary', 'Scornful', 'Screaming', 'Selfish', 'Serious', 'Shady',
+    'Shaky', 'Sharp', 'Shiny', 'Shy', 'Simple', 'Sleepy', 'Slow', 'Sly',
+    'Small', 'Smart', 'Smelly', 'Smiling', 'Smooth', 'Smug', 'Sober', 'Soft',
+    'Solemn', 'Square', 'Square', 'Steady', 'Strange', 'Strong', 'Stunning',
+    'Subjective', 'Successful', 'Surly', 'Sweet', 'Tactful', 'Tense',
+    'Thoughtful', 'Tight', 'Tiny', 'Tolerant', 'Uneasy', 'Unique', 'Unseen',
+    'Warm', 'Weak', 'Weird', 'WellCooked', 'Wild', 'Wise', 'Witty', 'Wonderful',
+    'Worried', 'Yellow', 'Young', 'Zealous'
+];
+
+/**
+ * Maps a string (category name) to the array of words from that category.
+ * @const
+ **/
+const CATEGORIES = {
+    '_PLURALNOUN_': PLURAL_NOUNS,
+    '_VERB_': VERBS,
+    '_ADVERB_': ADVERBS,
+    '_ADJECTIVE_': ADJECTIVES
+};
+
+/**
+* The list of available patterns for new room names generating.
+* @const
+**/
+const PATTERNS = [
+    '_ADJECTIVE__PLURALNOUN__VERB__ADVERB_'
+];
+
+/**
+ * Singleton object responsible for generating of new room names.
+ *
+ * @singleton
+ * @const
+ **/
+export const RoomnameGenerator = {
```

Comment:
RoomnameGenerator is a collection of functions without state. Consequently, I see no strong reason for RoomnameGenerator to exist in ES6 source code and be exported as such. Please remove RoomnameGenerator and export each of its functions separately.